### PR TITLE
Show time records

### PR DESCRIPTION
**Pull request by wontheone1**
Show time records

### CODE REVIEW

File: src/clj/hello_world/core.clj [hunk 1]
```diff
@@ -61,6 +63,9 @@
     (do
       (reset! game-start-time nil)
       (reset! sprites-state nil)
+      (swap! ranking (fn [ranking]
+                       (sort (conj ranking ?data))))
+      (println @ranking)
```
Comment (flyjwayur) [inline src/clj/hello_world/core.clj:69]: Where it is gonna print out 'ranking'? :) I wasnt able to see either in terminal and REPL

File: src/cljs/hello_world/game.cljs [hunk 2]
```diff
@@ -84,6 +84,7 @@
                                     send-puzzle-complete-fn!
                                     send-start-timer-fn!]}]
   (.setTo (.-scale (:play-button @util/game-state)) 0 0)
+  (.setTo (.-scale (:see-ranking-button @util/game-state)) 0 0)
```
Comment (flyjwayur) [inline src/cljs/hello_world/game.cljs]: Base on this project, I made it a bit differently :) https://github.com/flyjwayur/aikakoneMatka-puzzle/pull/17/commits/8aae12803adf843103fdb5f0fc1363dfcc9a3440

I tried to use 'currently-playing-game' in util ns :D

File: src/clj/hello_world/core.clj (hunk 1 above)
Comment (wontheone1) [reply src/clj/hello_world/core.clj:69]: 레플에 되지싶은뎅

File: src/cljs/hello_world/util.cljs [hunk 3]
```diff
@@ -66,6 +69,9 @@
 (defn- show-see-ranking-button! []
   (.setTo (.-scale (:see-ranking-button @game-state)) 0.5 0.5))
 
+(defn hide-see-ranking-button! []
```
Comment (flyjwayur) [inline src/cljs/hello_world/util.cljs]: 'defn- ' for show-see-ranking-button! and 'defn' for hide-see-ranking-button!. Why did it use 2 different form for each?

Comment (wontheone1) [reply src/cljs/hello_world/util.cljs]: `show-see-ranking-button!` is used in [the same ns](https://github.com/wontheone1/aikakone/pull/11/commits/b7d9b66b964b258f3b2d743140af93eb0eaf53a2#diff-af90a14acdfe6bf86ee3e8134539e257R80), `hide-see-ranking-button!` is used in [another ns](https://github.com/wontheone1/aikakone/pull/11/commits/b7d9b66b964b258f3b2d743140af93eb0eaf53a2#diff-f2b40445125147376abb7d8534bc16bfR87)

File: src/cljs/hello_world/game.cljs [hunk 4]
```diff
@@ -76,8 +80,30 @@
...
+    :play-button
+    (this-as this
+      (.button
+        (.-add @util/game)
+        10
+        10
+        "play-button"
+        (fn []
+          (util/destroy-stage-clear-text!)
+          (create-puzzle-board websocket-message-send-functions)
```
Comment (wontheone1) [inline src/cljs/hello_world/game.cljs:98]: @flyjwayur 여기서 `create-puzzle-board`가 정의 전에도 쓰이기 때문에 위에서 `declare` 했지요

**Pull request merged**